app/src/main/java/com/kks/nimblesurveyjetpackcompose/viewmodel/survey/SurveyDetailViewModel.kt

```diff
@@ -1,18 +1,37 @@
 package com.kks.nimblesurveyjetpackcompose.viewmodel.survey
 
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.kks.nimblesurveyjetpackcompose.di.IoDispatcher
+import com.kks.nimblesurveyjetpackcompose.model.ErrorModel
+import com.kks.nimblesurveyjetpackcompose.model.ResourceState
 import com.kks.nimblesurveyjetpackcompose.model.SurveyQuestion
```

Comment:
Don't forget we can prettify to clean up our imports 😉 Should be `cmd` + `option` + `l` 😄 
```suggestion
import com.kks.nimblesurveyjetpackcompose.model.*
```